Index: package/xorg/xkbcommon/xkbcommon.cache
===================================================================
--- package/xorg/xkbcommon/xkbcommon.cache	(revision 76475)
+++ package/xorg/xkbcommon/xkbcommon.cache	(working copy)
@@ -1,6 +1,6 @@
-[TIMESTAMP] 1739044198 Sat Feb  8 20:49:58 2025
-[BUILDTIME] 10 (5)
-[SIZE] 0.91 MB, 47 files
+[TIMESTAMP] 1745092253 Sat Apr 19 15:50:53 2025
+[BUILDTIME] 10 (9)
+[SIZE] 1.36 MB, 53 files

 [DEP] bash
 [DEP] binutils
@@ -11,6 +11,7 @@
 [OPT] doxygen
 [DEP] gawk
 [DEP] grep
+[DEP] gzip
 [DEP] icu4c
 [DEP] libxcb
 [DEP] libxml
@@ -23,4 +24,3 @@
 [OPT] wayland
 [DEP] wayland-protocols
 [OPT] xorg-server
-[DEP] xz
Index: package/xorg/xkbcommon/xkbcommon.desc
===================================================================
--- package/xorg/xkbcommon/xkbcommon.desc	(revision 76475)
+++ package/xorg/xkbcommon/xkbcommon.desc	(working copy)
@@ -1,6 +1,6 @@
 [COPY] --- T2-COPYRIGHT-BEGIN ---
 [COPY] t2/package/*/xkbcommon/xkbcommon.desc
-[COPY] Copyright (C) 2015 - 2025 The T2 SDE Project
+[COPY] Copyright (C) 2025 The T2 SDE Project
 [COPY] SPDX-License-Identifier: GPL-2.0
 [COPY] --- T2-COPYRIGHT-END ---

@@ -20,7 +20,7 @@

 [L] OpenSource
 [S] Stable
-[V] 1.7.0
+[V] 1.8.1
 [P] X -----5---9 112.600

 [O] . package/*/*/modular-x-conf.in
@@ -29,4 +29,4 @@
 [O] pkginstalled xorg-server || var_append mesonopt ' ' -Denable-x11=false

 [CV-URL] http://xkbcommon.org
-[D] 099af5b1843427941e5a7a4a6d176d5f5abeaa76f30cbd57db84875d libxkbcommon-1.7.0.tar.xz http://xkbcommon.org/download/
+[D] 7b131fefb6d972c106989488ad4e02ba8c573363bd260bd6c78f0d8e libxkbcommon-1.8.1.tar.gz https://github.com/xkbcommon/libxkbcommon/archive/refs/tags/xkbcommon-1.8.1/
