Index: package/wayland/wayvnc/wayvnc.cache
===================================================================
--- package/wayland/wayvnc/wayvnc.cache	(revision 76479)
+++ package/wayland/wayvnc/wayvnc.cache	(working copy)
@@ -1,16 +1,12 @@
+[TIMESTAMP] 1745127176 Sun Apr 20 01:32:56 2025
+[BUILDTIME] 4 (9)
+[SIZE] 0.22 MB, 10 files

-[TIMESTAMP] 1708948192 Mon Feb 26 12:49:52 2024
-[BUILDTIME] 6 (9)
-[SIZE] 0.17 MB, 8 files
-
-[DEP] 00-dirtree
 [DEP] aml
 [DEP] bash
 [DEP] binutils
-[DEP] cmake
 [DEP] coreutils
 [DEP] diffutils
-[DEP] findutils
 [DEP] gawk
 [DEP] git
 [DEP] grep
@@ -18,13 +14,14 @@
 [DEP] jansson
 [DEP] libdrm
 [DEP] linux-header
-[DEP] make
 [DEP] mesa
 [DEP] meson
 [DEP] neatvnc
 [DEP] ninja
+[DEP] openssl
 [DEP] pam
 [DEP] pixman
+[DEP] scdoc
 [DEP] sed
 [DEP] tar
 [DEP] wayland
Index: package/wayland/wayvnc/wayvnc.desc
===================================================================
--- package/wayland/wayvnc/wayvnc.desc	(revision 76479)
+++ package/wayland/wayvnc/wayvnc.desc	(working copy)
@@ -1,6 +1,6 @@
 [COPY] --- T2-COPYRIGHT-BEGIN ---
 [COPY] t2/package/*/wayvnc/wayvnc.desc
-[COPY] Copyright (C) 2023 - 2025 The T2 SDE Project
+[COPY] Copyright (C) 2025 The T2 SDE Project
 [COPY] SPDX-License-Identifier: GPL-2.0
 [COPY] --- T2-COPYRIGHT-END ---

@@ -22,6 +22,6 @@

 [L] MIT
 [S] Gamma
-[V] 0.8.0
+[V] 0.9.1

-[D] b9c77724a443defe1581b75f945dc030cef6ebc26a95248dd689f0f1 wayvnc-0.8.0.tar.gz https://github.com/any1/wayvnc/archive/refs/tags/v0.8.0/
+[D] 4ace2e45193f064d9a2e925c336a02e8e1ce77f306b04dd604989973 wayvnc-0.9.1.tar.gz https://github.com/any1/wayvnc/archive/refs/tags/v0.9.1/
