Index: package/wayland/foot/foot.desc
===================================================================
--- package/wayland/foot/foot.desc	(revision 76475)
+++ package/wayland/foot/foot.desc	(working copy)
@@ -1,6 +1,6 @@
 [COPY] --- T2-COPYRIGHT-BEGIN ---
 [COPY] t2/package/*/foot/foot.desc
-[COPY] Copyright (C) 2021 - 2025 The T2 SDE Project
+[COPY] Copyright (C) 2025 The T2 SDE Project
 [COPY] SPDX-License-Identifier: GPL-2.0
 [COPY] --- T2-COPYRIGHT-END ---

@@ -22,9 +22,9 @@

 [L] MIT
 [S] Stable
-[V] 1.20.2
+[V] 1.22.0
 [P] X -----5---9 160.000

 [O] var_append mesonopt ' ' '--buildtype=release -Dterminfo=disabled -Db_lto=false'

-[D] 16ed8bcd30dba7ef37cb8774c42ebefd62fb545519ce2880755d0c82 foot-1.20.2.tar.gz !https://codeberg.org/dnkl/foot/archive/1.20.2.tar.gz
+[D] b4a9cebf6001d5a3443b83955560a07f03e77b73af7b1a63406d5ed9 foot-1.22.0.tar.gz !https://codeberg.org/dnkl/foot/archive/1.22.0.tar.gz
