Index: package/graphic/libopenraw/libopenraw.cache
===================================================================
--- package/graphic/libopenraw/libopenraw.cache	(revision 76479)
+++ package/graphic/libopenraw/libopenraw.cache	(working copy)
@@ -1,21 +1,26 @@
+[TIMESTAMP] 1745138156 Sun Apr 20 04:35:56 2025
+[BUILDTIME] 100 (9)
+[SIZE] 19.18 MB, 37 files

-[TIMESTAMP] 1168080395 Sat Jan  6 11:46:35 2007
-[BUILDTIME] 5 (9)
-[SIZE] 0.42 MB, 22 files
-
-[DEP] 00-dirtree
+[DEP] bash
 [DEP] binutils
 [DEP] boost
+[DEP] bzip2
+[DEP] cargo
 [DEP] coreutils
+[DEP] curl
 [DEP] diffutils
-[DEP] file
 [DEP] findutils
 [DEP] gawk
+[DEP] gdk-pixbuf
+[DEP] glib
 [DEP] grep
 [DEP] libjpeg
+[DEP] libxml
 [DEP] linux-header
 [DEP] make
+[DEP] openssl
+[DEP] rustc
 [DEP] sed
-[DEP] sysfiles
 [DEP] tar
-[DEP] util-linux
+[DEP] tbb
Index: package/graphic/libopenraw/libopenraw.desc
===================================================================
--- package/graphic/libopenraw/libopenraw.desc	(revision 76479)
+++ package/graphic/libopenraw/libopenraw.desc	(working copy)
@@ -1,6 +1,6 @@
 [COPY] --- T2-COPYRIGHT-BEGIN ---
 [COPY] t2/package/*/libopenraw/libopenraw.desc
-[COPY] Copyright (C) 2006 - 2025 The T2 SDE Project
+[COPY] Copyright (C) 2025 The T2 SDE Project
 [COPY] SPDX-License-Identifier: GPL-2.0
 [COPY] --- T2-COPYRIGHT-END ---

@@ -19,6 +19,6 @@

 [L] LGPL
 [S] Stable
-[V] 0.0.8
+[V] 0.3.7

-[D] 2567334a3a0c050fd77145249bb8918b13299048ea0462ada295bc2b libopenraw-0.0.8.tar.gz http://libopenraw.freedesktop.org/download/
+[D] 7acac18317fed7dc8ca30b8a2cf824a6a541111bb946ceeb33ca41b6 libopenraw-0.3.7.tar.bz2 https://libopenraw.freedesktop.org/download/libopenraw-0.3.7.tar.bz2
