Index: package/wayland/sfwbar/sfwbar.cache
===================================================================
--- package/wayland/sfwbar/sfwbar.cache	(revision 76479)
+++ package/wayland/sfwbar/sfwbar.cache	(working copy)
@@ -1,6 +1,6 @@
-[TIMESTAMP] 1739213730 Mon Feb 10 19:55:30 2025
-[BUILDTIME] 12 (5)
-[SIZE] 1.14 MB, 185 files
+[TIMESTAMP] 1745110340 Sat Apr 19 20:52:20 2025
+[BUILDTIME] 100 (9)
+[SIZE] 1.24 MB, 203 files

 [DEP] alsa-lib
 [DEP] at-spi2-core
@@ -7,9 +7,9 @@
 [DEP] bash
 [DEP] binutils
 [DEP] cairo
+[DEP] cmake
 [DEP] coreutils
 [DEP] diffutils
-[DEP] docutils
 [DEP] gawk
 [DEP] gdk-pixbuf
 [DEP] glib
@@ -19,7 +19,6 @@
 [DEP] gzip
 [DEP] harfbuzz
 [DEP] json-c
-[DEP] libmpdclient
 [DEP] linux-header
 [DEP] meson
 [DEP] ninja
Index: package/wayland/sfwbar/sfwbar.desc
===================================================================
--- package/wayland/sfwbar/sfwbar.desc	(revision 76479)
+++ package/wayland/sfwbar/sfwbar.desc	(working copy)
@@ -1,6 +1,6 @@
 [COPY] --- T2-COPYRIGHT-BEGIN ---
 [COPY] t2/package/*/sfwbar/sfwbar.desc
-[COPY] Copyright (C) 2022 - 2025 The T2 SDE Project
+[COPY] Copyright (C) 2025 The T2 SDE Project
 [COPY] SPDX-License-Identifier: GPL-2.0
 [COPY] --- T2-COPYRIGHT-END ---

@@ -20,6 +20,6 @@

 [L] GPL3
 [S] Stable
-[V] 1.0_beta15
+[V] 1.0_beta16

-[D] c5b1834cfc9695151acb05d63953fdf39ee61870a7e589b3514f1ea2 sfwbar-1.0_beta15.tar.gz https://github.com/LBCrion/sfwbar/archive/v1.0_beta15/
+[D] 48fe16fe80f36396d4616f2694f33299a17fdea2dcc3eea0d1650315 sfwbar-1.0_beta16.tar.gz https://github.com/LBCrion/sfwbar/archive/refs/tags/v1.0_beta16/
